app/assets/javascripts/mobile/mobile_comments.js

```diff
@@ -11,6 +11,8 @@
     initialize: function() {
       var self = this;
 
+      new Diaspora.MarkdownEditor(".comment_box");
```

Comment:
I'm not sure this is big deal. Furthermore, we aren't constructing the object without rendering it. So that's a bit redundant.